remc2/utilities/BitmapIO.h

```diff
@@ -20,7 +20,7 @@ class BitmapIO
 
 	static void WriteImageBufferAsImageBMP(char* path, int width, int height, uint8_t* ptrPalette , uint8_t* ptrBuffer);
 	static void WriteRGBAImageBufferAsImageBMP(char* path, int width, int height, uint8_t* ptrPalette, uint8_t* ptrBuffer);
-	static void WritePaletteAsImageBMP(char* path, int numColors, uint8_t* ptrPalette);
+	static void WritePalletteAsImageBMP(char* path, int numColors, uint8_t* ptrPalette);
```

Comment:
It clearly has one "l". I got confused and thought someone else put the double "l" in there. Anyway, now the double l is everywhere and it's best to globally rename it to single l.

Reply:
Now I'll wait to see what you have to say about my modifications. In the meantime, I'll play with the network.
I've got some minor ROL/ROR optimization ready for the renderers. Subjectively it will add 1-2FPS.
inline uint32 __ROL4_16__(uint32 value) { return (value << 16) | (value >> 16); }
But I'll put that one in with other changes. We could also get rid of __ROL4_16__ altogether, it's about swapping WORDS in DWORD.

Reply:
Welcome back. I hope that you have a mild infection and get well soon.

I stumbled across the ROL before, because the sanitzer does not like it when the shifts loose bits. It does not realize that in this case it is intentional. It would be better to mask with `0xffff0000` and `0x0000ffff`.

In preliminary tests I also had problems with cursor position. But this might be due to running on a 4k notebook display on 200% scaling which essentially scales the SDL scaled image again. I need to look into this again.